src/components/Popups/ReplyField/MentionItem.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
+import { CompositeDecorator, ContentBlock, ContentState } from 'draft-js';
+
+type Props = {
+    children: React.ReactNode;
+    contentState: ContentState;
+    decoratedText: string;
+    entityKey?: string;
+};
+
+const MentionItem = ({ contentState, entityKey, children }: Props): JSX.Element => {
+    const id = entityKey ? contentState.getEntity(entityKey).getData().id : '';
+
+    return (
+        <a href={`/profile/${id}`} style={{ color: bdlBoxBlue }}>
+            {children}
+        </a>
+    );
+};
+
+export const mentionStrategy = (
+    contentBlock: ContentBlock,
+    callback: (start: number, end: number) => void,
+    contentState: ContentState,
+): void => {
+    contentBlock.findEntityRanges((character: { getEntity: () => string }) => {
+        const entityKey = character.getEntity();
+        return entityKey !== null && contentState.getEntity(entityKey).getType() === 'MENTION';
+    }, callback);
+};
+
+export const mentionDecorator = new CompositeDecorator([
```

Comment:
We should avoid having more than one "component" per file. This looks to be a HOC, essentially.